addons/binding/org.openhab.binding.autelis/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,568 @@
+		<label>Spa Temp</label>
+		<description>The current spa temperature. Note: Only accurate when spa is running</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="temp-airtemp">
+		<item-type>Number</item-type>
+		<label>Air Temp</label>
+		<description>The current air temperature</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="temp-tempunits">
+		<item-type>String</item-type>
+		<label>Temp Units</label>
+		<description>The selected units for temperature</description>
+		<state readOnly="true">
+			<options>
+				<option value="F">Fahrenheit</option>
+				<option value="C">Celsius</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="temp-htpump">
+		<item-type>Number</item-type>
+		<label>Heat Pump</label>
+		<description>Heat Pump</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump1">
+		<item-type>String</item-type>
+		<label>Pump 1</label>
+		<description>Pump 1</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump2">
+		<item-type>String</item-type>
+		<label>Pump 2</label>
+		<description>Pump 2</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump3">
+		<item-type>String</item-type>
+		<label>Pump 3</label>
+		<description>Pump 3</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump4">
+		<item-type>String</item-type>
+		<label>Pump 4</label>
+		<description>Pump 4</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump5">
+		<item-type>String</item-type>
+		<label>Pump 5</label>
+		<description>Pump 5</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump6">
+		<item-type>String</item-type>
+		<label>Pump 6</label>
+		<description>Pump 6</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump7">
+		<item-type>String</item-type>
+		<label>Pump 7</label>
+		<description>Pump 7</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="pump-pump8">
+		<item-type>String</item-type>
+		<label>Pump 8</label>
+		<description>Pump 8</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-chloren">
+		<item-type>Number</item-type>
+		<label>Chlorine</label>
+		<description>Chlorine</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-poolsp">
+		<item-type>Number</item-type>
+		<label>Chlorine Pool Setpoint</label>
+		<description>Chlorine Pool Setpoint</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="chlor-spasp">
+		<item-type>Number</item-type>
+		<label>Chlorine Spa Setpoint</label>
+		<description>Chlorine Sap Setpoint</description>
+		<state readOnly="true">
```

Comment:
Yes, this might be something they add later, or it might be a safety issue that they require you to change it at the equipment ( no chlorine == unsafe pool)